docs/api/filter-gateway.md

```diff
@@ -0,0 +1,1575 @@
+# Vald Filter Gateway APIs
+
+## Overview
+
+Filter Servie is responsible for providing insert, update, upsert and search interface for `Vald Filter Gateway`.
```

Comment:
I think it would be better to show more specific about when to use this API

Reply:
fixed :)